src/index.html

```diff
@@ -6,8 +6,278 @@
+            <div class="category__picture-wrapper">
+              <img 
+                class="category__picture" 
+                src="images/category1.png" 
+                alt="girl"
+              >
+            </div>
+            <div class="category__picture-wrapper">
+              <img 
+                class="category__picture" 
+                src="images/category2.png" 
+                alt="girl"
+              >
+            </div>
+            <h3 class="category__description">Headphones & earbuds</h3>
+          </div>
+
+          <div class="category__item category__item--speaker">
+            <div class="category__picture-wrapper">
+              <img 
+                class="category__picture" 
+                src="images/category3.png" 
+                alt="speaker"
+              >
+            </div>
+            <div class="category__picture-wrapper">
+              <img 
+                class="category__picture" 
+                src="images/category4.png" 
+                alt="speaker"
+              >
+            </div>
+            <h3 class="category__description">Speakers</h3>
+          </div>
+
+          <div class="category__item category__item--glass">
+            <div class="category__picture-wrapper">
+              <img 
+                class="category__picture" 
+                src="images/category5.png" 
+                alt="sunglass"
+              >
+            </div>
+            <div class="category__picture-wrapper">
+              <img 
+                class="category__picture" 
+                src="images/category6.png" 
+                alt="sunglass"
+              >
+            </div>
+            <h3 class="category__description">Audio sunglasses</h3>
+          </div>
+          <div class="category__button-wrapper">
+            <input 
+              class="category__button" 
+              type="button" 
+              value="view all"
+            >
+          </div>
+        </section>
+        <section class="content__why why">
+          <h2 class="why__header">Why buy direct from Bose</h2>
+          <div class="why__reasons">
+            <h4 class="why__reason">
+              Free 2-day shipping and returns
+            </h4>
+            <h4 class="why__reason">
+              90-day risk-free trial
+            </h4>
+            <h4 class="why__reason">
+              World class customer service
+            </h4>
+            <h4 class="why__reason">
+              My Bose account management
+            </h4>
+            <p>
+              A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+            </p>
+          </div>
+        </section>
+        <section class="content__contact contact">
+          <h2 class="contact__header">Contact us</h2>
+          <form 
```

Comment:
This section is also not aligned properly
may be try `justify-content: center;`
<img width="1187" alt="Screenshot 2022-07-26 at 2 24 05 PM" src="https://user-images.githubusercontent.com/100297341/180967525-4a95172e-697d-4d62-8856-a68333195656.png">